### ISSUE TITLE
Json Syntax Error on Appmesh Grafana Dashboard Canary Releases

### ISSUE THREAD

**Issue by kferrone** [aws/eks-charts#100]
Within the appmesh-grafana charts dashboards there is one called canary-releases.json with some json errors.  These are simple syntax issues.

On line 1179 there is an extra style key:
```json
  "style": "light",
  "style": "light",
```

On line 1182 there is a missing comma:
```json
  "tags": [
    "flagger"
  ]
  "templating": { 
```

**Comment by kferrone** [comment]
Soooooo . . . I guess y'all are cool with simple to fix typos. I'm guessing this is another AWS project meant for the graveyard with no replies. Ghosted . . .

**Comment by jaypipes** [comment]
Fixed and merged in release v0.0.19!
